suite-native/ethereum-tokens/src/ethereumTokensSelectors.ts

```diff
@@ -81,6 +82,26 @@ export const selectEthereumTokenHasFiatRates = (
     return !!rates?.rate;
 };
 
+export const selectTransactionTokensWithFiatRates = memoizeWithArgs(
```

Comment:
I think name `selectTokensTransactions` would make more sense. I think we can even drop WithFiatRates which is obvious because we do it everywhere.

Reply:
I've removed this selector in the end. I moved the token filtering by fiat rates right to the transaction list selector. So the filtering is newly done only once and not individually for every token as was happening in original solution of this PR. fixed in 0bd164f80